packages/utils/src/hl7v2-notifications/convert-adt-to-fhir-local-example.ts

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable no-useless-escape */
+import * as dotenv from "dotenv";
+dotenv.config();
+// keep that ^ on top
+import { Hl7Message } from "@medplum/core";
+import { convertHl7v2MessageToFhir } from "@metriport/core/command/hl7v2-subscriptions/hl7v2-to-fhir-conversion/index";
+import { getPatientIdsOrFail } from "@metriport/core/command/hl7v2-subscriptions/hl7v2-to-fhir-conversion/pid";
+import { errorToString } from "@metriport/shared";
+import fs from "fs";
+
+const filePath = "";
+const outputFolder = `${filePath}/converted`;
+const errorsFolder = `${filePath}/errors`;
+const fileName = "";
+
+/**
+ * Converts HL7v2 ADT messages to FHIR Bundle and saves them to a file.
```

Comment:
We could be more explicit on how to use this script, see `bulk-insert-patients.ts` and `coverage-assessment.ts`

Things I miss here:
- what is expected on `filePath` (is it a folder and it converts all files, is it a single file path?)
- what's the format of the file
   - single message, a message per line?
   - reference/example of the format of the file contents (could be a link to examples online?)

Also, nit: we usually set the script description after or before imports, so it's "above the fold" and not mingled w/ constants and smaller functions.